jenkins_node_scanner.py

```diff
@@ -165,6 +170,17 @@ def get_nodes(master):
         return []
 
 
+def test_port(ip_addr, port):
```

Comment:
IMHO, `is_port_open` would be a better name, since this function returns a boolean.